tools/generate_tables_sql_and_testNames.php

```diff
@@ -29,6 +29,7 @@
 require_once "../php/libraries/Database.class.inc";
 require_once "../php/libraries/NDB_Config.class.inc";
 require_once "../php/libraries/NDB_BVL_Instrument.class.inc";
+require_once "./generic_includes.php";
 
 $execute_mysql_query = ($argv[1] ?? true) === true;
```

Comment:
Shouldn't this just be `$execute_mysql_query = $argv[1] ?? true`?

Reply:
I think not because it's passing in a string for the argument values.

Reply:
A value of `"true"` will be false in the logic you have here. Is that desired?